src/org/zaproxy/zap/extension/ascan/ActiveScanAPI.java

```diff
@@ -386,6 +400,38 @@ public ApiResponse handleApiAction(String name, JSONObject params) throws ApiExc
 		return ApiResponseElement.OK;
 	}
 
+	private AlertThreshold getAlertThreshold(JSONObject params) throws ApiException {
+		if (!isParamExists(params.getString(PARAM_ALERT_THRESHOLD))) {
```

Comment:
It should be checked that the parameter was really sent (by using `JSONObject.has(String)` or `JSONObject.containsKey(Object)`) before obtaining it's value. Otherwise, this statement (more precisely `params.getString(...)`) will lead to `JSONException` if the parameter was not sent (given that it's optional it's very likely to happen).